schemata_json_schema/src/Normalizer/json/SchemataSchemaNormalizer.php

```diff
@@ -22,19 +21,26 @@ class SchemataSchemaNormalizer extends NormalizerBase {
   /**
    * {@inheritdoc}
    */
-  public function normalize($entity, $format = NULL, array $context = array()) {
+  public function normalize($entity, $format = NULL, array $context = []) {
+    $entity_type_id = $entity->getEntityTypeId();
+    $bundle = $entity->getBundleId();
     // Create the array of normalized fields, starting with the URI.
     /* @var $entity \Drupal\schemata\Schema\SchemaInterface */
+    $route_name = $bundle ?
+      sprintf('schemata.%s:%s', $entity_type_id, $bundle) :
+      sprintf('schemata.%s', $entity_type_id);
+    $generated_url = Url::fromRoute($route_name, [], ['absolute' => TRUE])
+      ->toString(TRUE);
```

Comment:
At the time I thought that it only added an intermediate step that could be avoided. I'll add it back.